flent/scripts/ss_iterate.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+#Copyright (C) 2016  Matthias Tafelmeier
+
+#ss_iterate.sh is free software: you can redistribute it and/or modify
+#it under the terms of the GNU General Public License as published by
+#the Free Software Foundation, either version 3 of the License, or
+#(at your option) any later version.
+
+#lindwurm is distributed in the hope that it will be useful,
+#but WITHOUT ANY WARRANTY; without even the implied warranty of
+#MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#GNU General Public License for more details.
+
+#You should have received a copy of the GNU General Public License
+#along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+count=10
+interval=0.1
+host=localhost
+
+while getopts "c:I:H:t:" opt; do
+    case $opt in
+        c) count=$OPTARG ;;
+        I) interval=$OPTARG ;;
+        H) host=$OPTARG ;;
+        t) target=$OPTARG ;;
```

Comment:
Fair enough, let me introduce a usage predicate. Swallowing an err by defaults here is even more a dread in the direction of debugging.